### DISCUSSION TITLE
Detect fraud and compliance with procurement criteria

### DISCUSSION THREAD

**Discussion by eprocurementontology** [eprocurementontology/eprocurementontology#508] [Ideas]
In the second working group meeting, a new use case was proposed: "Detect fraud and compliance with procurement criteria" (i.e. use case 1.4 in the [landscaping document](https://joinup.ec.europa.eu/node/159724))

This issue will group the input received regarding this use case. Please share your input below.

**Comment by brechtwyns** [comment]
Thanks to @JachymHercher for sharing the following:
https://opentender.eu/blog/2017-03-towards-more-transparency/
https://opentender.eu/blog/assets/downloads/digiwhist_implementers_guide.pdf (page 18)

**Comment by brechtwyns** [comment]
The landscaping document describes this use case as follows:
_For assuring efficiency and transparency, and for detecting fraud and corruption in public administrations, EU institutions and contracting authorities, rigorous audits of procurement take place. In order to improve and further automate the audit process, different data should be made available in structured, machine-readable formats so different data sources can be referenced and integrated. The creation of the ePO will be a first step towards achieving such integration._

**Example:** 
While auditing the evidences submitted by the tenderer who was awarded the contract, the auditor noticed that the supplier did not comply with the location criteria that were agreed during the signing of the contract. The payment evidence adduced, proved that besides the initial agreement, the supplier leased services outside the European Union to reduce the cost of the works. Publishing e-Procurement data in a structured, linked and machine-readable format, allows the interconnection of data on transactions, criteria, contracts and evidences from different sources, e.g. including also BRIS and ECRIS, thus facilitating cross-checking and automated fraud detection. 

**Information requirements:**
In this case it would be required:
- Of the ePO to model the evidences, the contract, the procurement criteria, including the location criteria;
- To link the data from the ePO to data in other datasets, such as procurement systems of different countries or the BRIS or ECRIS.
